content/docs/features.md

```diff
@@ -24,3 +24,29 @@ spec:
     deploy:
     ...
 ```
+
+## EXPERIMENTAL - Validating admission webhook
+
+In 0.9.0 version of KUDO we introduced a new experimental feature - validating admission webhook. When enabled, it helps to enforce consistency of our CRDs and it also makes sure that the execution plan is atomic and deterministic.
+
+### Why we need this?
+
+Validating [admission webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/) is HTTP callback that receive admission request (from API server) and let you apply validation rules on them. This validation cannot be performed inside controller because that would happen AFTER the resource was stored in ETCD thus leaving us with illegal state already in. Kubernetes admission webhooks require HTTPS to be used for this endpoint.
```

Comment:
I think this sentence `Kubernetes admission webhooks require HTTPS to be used for this endpoint` can be moved to the section where we explain cert-manager dependency